dpc-sdp/ripple-framework
# [R20-293] theme values are able to be set from Tide site taxonomy

Moved site mapping to ripple-tide-api



**Issue**:

### What I did

- Moved site mapping to ripple-tide-api
- Added theme value in site mapping
- Made tide baseUrl come from CONTENT_API_SERVER env var - You will need to add a .env file in your local with CONTENT_API_SERVER value set

### How to test

- Copy examples/nuxt-app/.example.env to examples/nuxt-app/.env
- Update CONTENT_API_SERVER to https://nginx-php.pr-136.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au
- Go to https://nginx-php.pr-136.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au/taxonomy/term/8888/edit
- Update `rpl-clr-primary` in Site Theme values section
- Check site header updates

### Checklist




#### For all PR's


#### For new components only
